packages/eslint-plugin/lib/config/by-project-type/monorepo-root.ts

```diff
@@ -0,0 +1,9 @@
+import type { Linter } from "eslint";
+
+const config: Linter.Config = {
+    extends: [
+        "plugin:@workleap/typescript-library"
```

Comment:
Not a big deal but personally I would probably have duplicated the configs instead of extending typescript-library. Doing it this way feels like pushing the work of duplicating the configs to the next maintainer who might have to deal what differences between the "typescript-library" config and the "monorepo-root" config values.